README.md

```diff
@@ -155,17 +155,17 @@ A brief overview on how to configure and maintain an Iroha instance:
 - [Storage](#storage)
 - [Scalability](#scalability)
 
-## Configuration
+### Configuration
 
 You can provide configuration parameters either as a `config.json` or using environment variables. Refer to the [detailed list](./docs/source/references/config.md) of all available configuration parameters.
 
 Configuration example you may use as a reference point: [cli/src/samples.rs](./cli/src/samples.rs)
 
-## Endpoints
+### Endpoints
 
-You can find the detailed list of all available endpoints in the [API specifications](./docs/source/references/api_spec.md#endpoints).
+You can find the detailed list of all available endpoints in the [API Reference](https://hyperledger.github.io/iroha-2-docs/api/torii-endpoints).
```

Comment:
Try the following or something similar that you find fitting (i believe, it is a great idea to start off the sentence with stating the most important thing a user can find in the topic we're referencing):
`For a detailed list of all available endpoints and operations, see [API > Torii Endpoints](link-to-the-topic.md)`

A more detailed version:
`For a list of all endpoints, available operations, and ways to customize them with parameters, see [API > Torii Endpoints](link-to-the-topic.md)`

Reply:
Done